packages/shared/scope-hoisting/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "@parcel/scope-hoisting",
+  "version": "2.0.0",
+  "description": "Blazing fast, zero configuration web application bundler",
+  "main": "src/index.js",
+  "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/parcel-bundler/parcel.git"
+  },
+  "engines": {
+    "node": ">= 8.0.0"
+  },
+  "scripts": {
+    "pretest": "yarn build",
+    "test": "cross-env NODE_ENV=test mocha",
+    "test-ci": "yarn build && yarn test",
+    "build": "babel src -d lib",
+    "prepublish": "yarn build"
+  },
+  "devDependencies": {
+    "@parcel/eslint-config": "1.10.3",
+    "mocha": "^5.2.0"
+  },
+  "dependencies": {
+    "@babel/generator": "^7.3.3",
+    "@babel/template": "^7.2.2",
+    "@babel/traverse": "^7.2.3",
+    "@babel/types": "^7.3.3",
+    "@parcel/types": "^2.0.0",
```

Comment:
Is this a real runtime dependency? Otherwise could probably just remove it as it's also not necessary as a devdependency.